datamol/convert.py

```diff
@@ -248,14 +256,17 @@ def to_inchikey(mol: Union[str, dm.Mol]) -> Optional[str]:
     Args:
         mol: a molecule
     """
+    if isinstance(mol, str):
+        mol = dm.to_mol(mol)
 
     if mol is None:
         return None
 
-    if isinstance(mol, str):
-        mol = dm.to_mol(mol)
-
-    return Chem.MolToInchiKey(mol)
+    inchikey = Chem.MolToInchiKey(mol)
+    if not inchikey:
+        logger.warning("Inchikey computation failed, setting output to None")
```

Comment:
I am not a huge fan of the warnings here. We don't do it for the other `convert` functions, and it can also potentially pollute the logs quite quickly.

Do you mind removing those?

Reply:
(not related but FYI `loguru` does not play well with `multiprocessing` or `loky`)